### DISCUSSION TITLE
Better spam account filtering for COPTR

### DISCUSSION THREAD

**Discussion by anjackson** [digipres/digipres.github.io#36] [General]
Currently, signing up to COPTR means getting past a test that was intended to stop spammers signing up, but is way too exclusive/cliquey (see https://coptr.digipres.org/index.php?title=Special:CreateAccount&returnto=Main+Page)

> What short name does OAIS use for an information package that is used for submission? 

I mean, really, whose terrible idea was that?! (_it was me_)

Maybe we can do without it? Or use a ReCAPTCHA instead?

**Comment by ross-spencer** [comment]
It's better than File Format Wiki! :sweat_smile: 

+1 though, seems a fair idea to change for something more generic/accessible.

**Comment by anjackson** [comment]
Just to note that the spammers worked around this anyway, and COPTR has ended up drowning in spam. It is not really sustainable in it's current form. I'm working with @carlwilson to clean things up a bit, but really we need a different approach.

**Comment by ablwr** [comment]
This was one of the motivating factors for moving the [a/v artifact atlas](https://www.avartifactatlas.com/) from a wiki-base to github-based platform, where it now sits running on the Jekyll framework.

**Comment by anjackson** [comment]
Thanks @ross-spencer @ablwr - I'm pretty sure tweaking the current COPTR approach is off the table, so I've closed this and I'm attempting to start a more wide-ranging discussion at #53

**Comment by ross-spencer** [comment]
Thanks Andy, left a few thoughts on #53. Out of interest, did you and Carl look at options wrt NGINX deny-lists? I managed to get rid of nearly all spamming on one of my ActivityPub instances by blocking a few-user agents I saw most activity from in the access.log, especially Python/requests user-agents. There might be other approaches with specific endpoints.
